.vib/grafana/cypress/cypress/integration/grafana_test_suite.js

```diff
@@ -0,0 +1,146 @@
+/// <reference types="cypress" />
+import {
+  deleteDashboard,
+  deleteAPIKey,
+  clickDeleteButton,
+  verifySuccesOfAction
+} from './utils'
+
+it('allows user to log out', () => {
+  cy.login();
+  cy.visit('/logout');
+  cy.get('.login-content-box').should('exist');
+})
+
+it('allows creating a dashboard with a panel', () => {
+  const CLOSE_DIALOGUE_BUTTON = 'button[aria-label="Close dialogue"]';
```

Comment:
This is nice, but I'm having a hard time trying to tell when to use a constant or directly write the selector in the command. 

I'd say that it is fine to use constants when you will be reusing them through different lines, but this time it seems it is only used once.

Reply:
I wanted to add this constant for readability purposes. This way, when the time comes to change this locator due to an upgrade, we can easily find it. WDYT?

Reply:
You are right, it is better to maintain uniformity for now. I've made the changes in the spec. 
We will be using the constants directly unless there is a need for reuse of the same locator.